kooplex-proxy/proxy/crud.py

```diff
@@ -1,33 +1,74 @@
 from sqlalchemy.orm import Session
-from sqlalchemy import text
+from sqlalchemy.ext.declarative import DeclarativeMeta
 
 import models
-from database import engine
+from database import SCHEMA_SET
 
 
-def get_country_samples(db: Session, skip: int = 0, limit: int = 100):
-    return db.query(models.MViewCountrySamples).offset(skip).limit(limit).all()
+def schema_changing(model: DeclarativeMeta, endp_schema: str):
+    current_schema = model.get_schema()
+    if current_schema != endp_schema:
+        if endp_schema in SCHEMA_SET:
+            model.set_schema(schema_name=endp_schema)
+            print(f"-- Changing schema from '{current_schema}' to '{endp_schema}' for the endpoint: country_samples --")
+            return 0
+        else:
+            print(f"This schema is unavailable for using, please use one of these schemas: {SCHEMA_SET}")
```

Comment:
security exposure. You should only provide the nicknames / AKA dictionary keys in the error message.